### PR TITLE
Added QR Code Generator Program in Jarvis

### PR DESCRIPTION
**Pull request by vishalmaurya850**
Closes: 

- **Title** : QR code Generator
- **Name:** Vishal Maurya
- **Idenitfy yourself:** SSOC24 Contributor

### Describe the add-ons or changes you've made 📃

I have updated the requirement.txt file and added a qr code generator program integrated in Jarvis and also updated imageProgram.py file

### Type of change ☑️

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/Jarvis/blob/main/Contributing.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/Jarvis/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by vishalmaurya850** [comment]
Please merge my PR @Avdhesh-Varshney

**Review by vishalmaurya850 — COMMENTED** [review]
Merge it

File: requirements.txt
Comment (Avdhesh-Varshney) [inline requirements.txt:1]: resolve this error

File: src/apps/pages/programs/ImageGenerators/qr_code_generator.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import streamlit as st
+import qrcode
+from PIL import Image
+import io
+
+def main():
```
Comment (Avdhesh-Varshney) [inline src/apps/pages/programs/ImageGenerators/qr_code_generator.py:6]: rename this main to `qrCodeGenerator()`

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

File: requirements.txt
Comment (vishalmaurya850) [reply requirements.txt:1]: Please attach the error you are getting as there is no error at my side.

Comment (Avdhesh-Varshney) [reply requirements.txt:1]: attach a working video

**Comment by vishalmaurya850** [comment]
Uploading Screen Recording 2024-07-05 155532.mp4…

**Comment by vishalmaurya850** [comment]
https://github.com/Avdhesh-Varshney/Jarvis/assets/40622097/e837b959-63ce-43f7-b546-d087a965eb9d

**Comment by vishalmaurya850** [comment]
https://github.com/Avdhesh-Varshney/Jarvis/assets/40622097/f299ae8b-0b7a-4222-a845-0ce308538655

**Review by Avdhesh-Varshney — APPROVED** [review]

**Comment by Avdhesh-Varshney** [comment]
> Screen.Recording.2024-07-05.155846.mp4

@vishalmaurya850 only this video is required not the video for setup of project. And program LGTM.

**Comment by Avdhesh-Varshney** [comment]
@vishalmaurya850 mention your issue and properly link your issue. Otherwise labels will not attach on merging the pr

**Comment by vishalmaurya850** [comment]
Which Issue are you talking about @Avdhesh-Varshney

**Comment by Avdhesh-Varshney** [comment]
> Which Issue are you talking about @Avdhesh-Varshney

related to this pr. Properly mention your issue number in the pr template while raising not in the comment section after raising the pr.

**Comment by vishalmaurya850** [comment]
Ok now it's perfect??

**Pull request merged**